### PR TITLE
Fix claim URLs not been displayed in google outbound provisioning

### PR DESCRIPTION
**Pull request by dilin993**
Fix https://github.com/wso2/product-is/issues/4723
Fix https://github.com/wso2/product-is/issues/4721

### CODE REVIEW

File: components/idp-mgt/org.wso2.carbon.idp.mgt.ui/src/main/resources/web/idpmgt/idp-mgt-edit.jsp [hunk 1]
```diff
@@ -1467,30 +1467,30 @@
         jQuery('#claimAddTable .claimrow').each(function () {
             var val = htmlEncode($(this).val());
             if (val.trim() != "") {
-                if (val == '<%=Encode.forJavaScriptBlock(userIdClaimURI)%>') {
+                if (val == unescape('<%=Encode.forJavaScriptBlock(userIdClaimURI)%>')) {
```
Comment (NShani) [inline components/idp-mgt/org.wso2.carbon.idp.mgt.ui/src/main/resources/web/idpmgt/idp-mgt-edit.jsp]:
```suggestion
                if (val == unescape('<%=Encode.forJavaScript(userIdClaimURI)%>')) {
```

Comment (NShani) [inline components/idp-mgt/org.wso2.carbon.idp.mgt.ui/src/main/resources/web/idpmgt/idp-mgt-edit.jsp]:
```suggestion
                if (val == unescape('<%=Encode.forJavaScript(userIdClaimURI)%>')) {
```

File: components/idp-mgt/org.wso2.carbon.idp.mgt.ui/src/main/resources/web/idpmgt/idp-mgt-edit.jsp [hunk 2]
```diff
@@ -1467,30 +1467,30 @@
...
                     $user_id_claim_dropdown.append('<option selected="selected">' + val + '</option>');
                 } else {
                     $user_id_claim_dropdown.append('<option>' + val + '</option>');
                 }
-                if (val == '<%=Encode.forJavaScriptBlock(roleClaimURI)%>') {
+                if (val == unescape('<%=Encode.forJavaScriptBlock(roleClaimURI)%>')) {
```
Comment (NShani) [inline components/idp-mgt/org.wso2.carbon.idp.mgt.ui/src/main/resources/web/idpmgt/idp-mgt-edit.jsp]:
```suggestion
                if (val == unescape('<%=Encode.forJavaScript(roleClaimURI)%>')) {
```

File: components/idp-mgt/org.wso2.carbon.idp.mgt.ui/src/main/resources/web/idpmgt/idp-mgt-edit.jsp [hunk 3]
```diff
@@ -1467,30 +1467,30 @@
...
                     $role_claim_dropdown.append('<option selected="selected">' + val + '</option>');
                 } else {
                     $role_claim_dropdown.append('<option>' + val + '</option>');
                 }
 
-                if (val == '<%=Encode.forJavaScriptBlock(googlePrimaryEmailClaim)%>') {
+                if (val === unescape('<%=Encode.forJavaScriptBlock(googlePrimaryEmailClaim)%>')) {
```
Comment (NShani) [inline components/idp-mgt/org.wso2.carbon.idp.mgt.ui/src/main/resources/web/idpmgt/idp-mgt-edit.jsp]:
```suggestion
                if (val == unescape('<%=Encode.forJavaScript(googlePrimaryEmailClaim)%>')) {
```

File: components/idp-mgt/org.wso2.carbon.idp.mgt.ui/src/main/resources/web/idpmgt/idp-mgt-edit.jsp [hunk 4]
```diff
@@ -1467,30 +1467,30 @@
...
                     $google_prov_email_claim_dropdown.append('<option selected="selected">' + val + '</option>');
                 } else {
                     $google_prov_email_claim_dropdown.append('<option>' + val + '</option>');
                 }
 
-                if (val == '<%=Encode.forJavaScriptBlock(googleFamilyNameClaim)%>') {
+                if (val == unescape('<%=Encode.forJavaScriptBlock(googleFamilyNameClaim)%>')) {
```
Comment (NShani) [inline components/idp-mgt/org.wso2.carbon.idp.mgt.ui/src/main/resources/web/idpmgt/idp-mgt-edit.jsp]:
```suggestion
                if (val == unescape('<%=Encode.forJavaScript(googleFamilyNameClaim)%>')) {
```

File: components/idp-mgt/org.wso2.carbon.idp.mgt.ui/src/main/resources/web/idpmgt/idp-mgt-edit.jsp [hunk 5]
```diff
@@ -1467,30 +1467,30 @@
...
                     $google_prov_familyname_claim_dropdown.append('<option selected="selected">' + val + '</option>');
                 } else {
                     $google_prov_familyname_claim_dropdown.append('<option>' + val + '</option>');
                 }
 
-                if (val == '<%=Encode.forJavaScriptBlock(googleGivenNameClaim)%>') {
+                if (val == unescape('<%=Encode.forJavaScriptBlock(googleGivenNameClaim)%>')) {
```
Comment (NShani) [inline components/idp-mgt/org.wso2.carbon.idp.mgt.ui/src/main/resources/web/idpmgt/idp-mgt-edit.jsp]:
```suggestion
                if (val == unescape('<%=Encode.forJavaScript(googleGivenNameClaim)%>')) {
```

File: components/idp-mgt/org.wso2.carbon.idp.mgt.ui/src/main/resources/web/idpmgt/idp-mgt-edit.jsp [hunk 6]
```diff
@@ -1534,10 +1534,10 @@
 
             <% for(int i =0 ; i< claimUris.length ; i++){
 
-           		 if(claimUris[i].equals(Encode.forJavaScriptBlock(googlePrimaryEmailClaim))){  %>
-            google_prov_email_option += '<option  selected="selected" value="' + "<%=claimUris[i]%>" + '">' + "<%=claimUris[i]%>" + '</option>';
+           		 if(claimUris[i].equals(googlePrimaryEmailClaim)){  %>
+            google_prov_email_option += '<option  selected="selected" value="' + "<%=Encode.forHtmlAttribute(claimUris[i])%>" + '">' + "<%=Encode.forHtmlContent(claimUris[i])%>" + '</option>';
```
Comment (NShani) [inline components/idp-mgt/org.wso2.carbon.idp.mgt.ui/src/main/resources/web/idpmgt/idp-mgt-edit.jsp]:
```suggestion
            google_prov_email_option += '<option  selected="selected" value="' + "<%=Encode.forHtml(claimUris[i])%>" + '">' + "<%=Encode.forHtml(claimUris[i])%>" + '</option>';
```

File: components/idp-mgt/org.wso2.carbon.idp.mgt.ui/src/main/resources/web/idpmgt/idp-mgt-edit.jsp [hunk 7]
```diff
@@ -1534,10 +1534,10 @@
...
             <% 	 } else {  %>
-            google_prov_email_option += '<option value="' + "<%=claimUris[i]%>" + '">' + "<%=claimUris[i]%>" + '</option>';
+            google_prov_email_option += '<option value="' + "<%=Encode.forHtmlAttribute(claimUris[i])%>" + '">' + "<%=Encode.forHtmlContent(claimUris[i])%>" + '</option>';
```
Comment (NShani) [inline components/idp-mgt/org.wso2.carbon.idp.mgt.ui/src/main/resources/web/idpmgt/idp-mgt-edit.jsp]:
```suggestion
            google_prov_email_option += '<option value="' + "<%=Encode.forHtml(claimUris[i])%>" + '">' + "<%=Encode.forHtml(claimUris[i])%>" + '</option>';
```

File: components/idp-mgt/org.wso2.carbon.idp.mgt.ui/src/main/resources/web/idpmgt/idp-mgt-edit.jsp [hunk 8]
```diff
@@ -1546,10 +1546,10 @@
 
             <% for(int i =0 ; i< claimUris.length ; i++){
 
-           		 if(claimUris[i].equals(Encode.forJavaScriptBlock(googleFamilyNameClaim))){  %>
-            google_prov_family_email_option += '<option  selected="selected" value="' + "<%=claimUris[i]%>" + '">' + "<%=claimUris[i]%>" + '</option>';
+           		 if(claimUris[i].equals(googleFamilyNameClaim)){  %>
+            google_prov_family_email_option += '<option  selected="selected" value="' + "<%=Encode.forHtmlAttribute(claimUris[i])%>" + '">' + "<%=Encode.forHtmlContent(claimUris[i])%>" + '</option>';
```
Comment (NShani) [inline components/idp-mgt/org.wso2.carbon.idp.mgt.ui/src/main/resources/web/idpmgt/idp-mgt-edit.jsp]:
```suggestion
            google_prov_family_email_option += '<option  selected="selected" value="' + "<%=Encode.forHtml(claimUris[i])%>" + '">' + "<%=Encode.forHtml(claimUris[i])%>" + '</option>';
```

File: components/idp-mgt/org.wso2.carbon.idp.mgt.ui/src/main/resources/web/idpmgt/idp-mgt-edit.jsp [hunk 9]
```diff
@@ -1546,10 +1546,10 @@
...
             <% 	 } else {  %>
-            google_prov_family_email_option += '<option value="' + "<%=claimUris[i]%>" + '">' + "<%=claimUris[i]%>" + '</option>';
+            google_prov_family_email_option += '<option value="' + "<%=Encode.forHtmlAttribute(claimUris[i])%>" + '">' + "<%=Encode.forHtmlContent(claimUris[i])%>" + '</option>';
```
Comment (NShani) [inline components/idp-mgt/org.wso2.carbon.idp.mgt.ui/src/main/resources/web/idpmgt/idp-mgt-edit.jsp]:
```suggestion
            google_prov_family_email_option += '<option value="' + "<%=Encode.forHtml(claimUris[i])%>" + '">' + "<%=Encode.forHtml(claimUris[i])%>" + '</option>';
```

File: components/idp-mgt/org.wso2.carbon.idp.mgt.ui/src/main/resources/web/idpmgt/idp-mgt-edit.jsp [hunk 10]
```diff
@@ -1558,16 +1558,16 @@
 
             <% for(int i =0 ; i< claimUris.length ; i++){
 
-           		 if(claimUris[i].equals(Encode.forJavaScriptBlock(googleGivenNameClaim))){  %>
-            google_prov_givenname_option += '<option  selected="selected" value="' + "<%=claimUris[i]%>" + '">' + "<%=claimUris[i]%>" + '</option>';
+           		 if(claimUris[i].equals(googleGivenNameClaim)){  %>
+            google_prov_givenname_option += '<option  selected="selected" value="' + "<%=Encode.forHtmlAttribute(claimUris[i])%>" + '">' + "<%=Encode.forHtmlContent(claimUris[i])%>" + '</option>';
```
Comment (NShani) [inline components/idp-mgt/org.wso2.carbon.idp.mgt.ui/src/main/resources/web/idpmgt/idp-mgt-edit.jsp]:
```suggestion
            google_prov_givenname_option += '<option  selected="selected" value="' + "<%=Encode.forHtml(claimUris[i])%>" + '">' + "<%=Encode.forHtml(claimUris[i])%>" + '</option>';
```

File: components/idp-mgt/org.wso2.carbon.idp.mgt.ui/src/main/resources/web/idpmgt/idp-mgt-edit.jsp [hunk 11]
```diff
@@ -1558,16 +1558,16 @@
...
             <% 	 } else {  %>
-            google_prov_givenname_option += '<option value="' + "<%=claimUris[i]%>" + '">' + "<%=claimUris[i]%>" + '</option>';
+            google_prov_givenname_option += '<option value="' + "<%=Encode.forHtmlAttribute(claimUris[i])%>" + '">' + "<%=Encode.forHtmlContent(claimUris[i])%>" + '</option>';
```
Comment (NShani) [inline components/idp-mgt/org.wso2.carbon.idp.mgt.ui/src/main/resources/web/idpmgt/idp-mgt-edit.jsp]:
```suggestion
            google_prov_givenname_option += '<option value="' + "<%=Encode.forHtml(claimUris[i])%>" + '">' + "<%=Encode.forHtml(claimUris[i])%>" + '</option>';
```

File: components/idp-mgt/org.wso2.carbon.idp.mgt.ui/src/main/resources/web/idpmgt/idp-mgt-edit.jsp [hunk 12]
```diff
@@ -1558,16 +1558,16 @@
...
             <%	 }
             }%>
 
 
             <% for(int i =0 ; i< claimUris.length ; i++){%>
-            option += '<option value="' + "<%=claimUris[i]%>" + '">' + "<%=claimUris[i]%>" + '</option>';
+            option += '<option value="' + "<%=Encode.forHtmlAttribute(claimUris[i])%>" + '">' + "<%=Encode.forHtmlContent(claimUris[i])%>" + '</option>';
```
Comment (NShani) [inline components/idp-mgt/org.wso2.carbon.idp.mgt.ui/src/main/resources/web/idpmgt/idp-mgt-edit.jsp]:
```suggestion
            option += '<option value="' + "<%=Encode.forHtml(claimUris[i])%>" + '">' + "<%=Encode.forHtml(claimUris[i])%>" + '</option>';
```

File: components/idp-mgt/org.wso2.carbon.idp.mgt.ui/src/main/resources/web/idpmgt/idp-mgt-edit.jsp (hunk 1 above)
Comment (dilin993) [reply components/idp-mgt/org.wso2.carbon.idp.mgt.ui/src/main/resources/web/idpmgt/idp-mgt-edit.jsp]: I will change this

File: components/idp-mgt/org.wso2.carbon.idp.mgt.ui/src/main/resources/web/idpmgt/idp-mgt-edit.jsp (hunk 2 above)
Comment (dilin993) [reply components/idp-mgt/org.wso2.carbon.idp.mgt.ui/src/main/resources/web/idpmgt/idp-mgt-edit.jsp]: I will change this

File: components/idp-mgt/org.wso2.carbon.idp.mgt.ui/src/main/resources/web/idpmgt/idp-mgt-edit.jsp (hunk 3 above)
Comment (dilin993) [reply components/idp-mgt/org.wso2.carbon.idp.mgt.ui/src/main/resources/web/idpmgt/idp-mgt-edit.jsp]: I will change this

File: components/idp-mgt/org.wso2.carbon.idp.mgt.ui/src/main/resources/web/idpmgt/idp-mgt-edit.jsp (hunk 4 above)
Comment (dilin993) [reply components/idp-mgt/org.wso2.carbon.idp.mgt.ui/src/main/resources/web/idpmgt/idp-mgt-edit.jsp]: I will change this

File: components/idp-mgt/org.wso2.carbon.idp.mgt.ui/src/main/resources/web/idpmgt/idp-mgt-edit.jsp (hunk 5 above)
Comment (dilin993) [reply components/idp-mgt/org.wso2.carbon.idp.mgt.ui/src/main/resources/web/idpmgt/idp-mgt-edit.jsp]: I will change this

File: components/idp-mgt/org.wso2.carbon.idp.mgt.ui/src/main/resources/web/idpmgt/idp-mgt-edit.jsp (hunk 6 above)
Comment (dilin993) [reply components/idp-mgt/org.wso2.carbon.idp.mgt.ui/src/main/resources/web/idpmgt/idp-mgt-edit.jsp]: I will change this

File: components/idp-mgt/org.wso2.carbon.idp.mgt.ui/src/main/resources/web/idpmgt/idp-mgt-edit.jsp (hunk 7 above)
Comment (dilin993) [reply components/idp-mgt/org.wso2.carbon.idp.mgt.ui/src/main/resources/web/idpmgt/idp-mgt-edit.jsp]: I will change this

File: components/idp-mgt/org.wso2.carbon.idp.mgt.ui/src/main/resources/web/idpmgt/idp-mgt-edit.jsp (hunk 8 above)
Comment (dilin993) [reply components/idp-mgt/org.wso2.carbon.idp.mgt.ui/src/main/resources/web/idpmgt/idp-mgt-edit.jsp]: I will change this

File: components/idp-mgt/org.wso2.carbon.idp.mgt.ui/src/main/resources/web/idpmgt/idp-mgt-edit.jsp (hunk 9 above)
Comment (dilin993) [reply components/idp-mgt/org.wso2.carbon.idp.mgt.ui/src/main/resources/web/idpmgt/idp-mgt-edit.jsp]: I will change this

File: components/idp-mgt/org.wso2.carbon.idp.mgt.ui/src/main/resources/web/idpmgt/idp-mgt-edit.jsp (hunk 10 above)
Comment (dilin993) [reply components/idp-mgt/org.wso2.carbon.idp.mgt.ui/src/main/resources/web/idpmgt/idp-mgt-edit.jsp]: I will change this

File: components/idp-mgt/org.wso2.carbon.idp.mgt.ui/src/main/resources/web/idpmgt/idp-mgt-edit.jsp (hunk 11 above)
Comment (dilin993) [reply components/idp-mgt/org.wso2.carbon.idp.mgt.ui/src/main/resources/web/idpmgt/idp-mgt-edit.jsp]: I will change this

File: components/idp-mgt/org.wso2.carbon.idp.mgt.ui/src/main/resources/web/idpmgt/idp-mgt-edit.jsp (hunk 12 above)
Comment (dilin993) [reply components/idp-mgt/org.wso2.carbon.idp.mgt.ui/src/main/resources/web/idpmgt/idp-mgt-edit.jsp]: I will change this

**Pull request merged**